### ISSUE TITLE
 TabbedPage is not a type

### ISSUE THREAD

**Issue by ruisebastiao** [lirios/fluid#46]
Hello,

i'm getting the following error running the demo:
```
QQmlApplicationEngine failed to load component
qrc:/main.qml:74 TabbedPage is not a type
```
what am i doing wrong

**Comment by IlyaBizyaev** [comment]
Same problem.

**Comment by plfiorini** [comment]
This should be fixed now

**Comment by AlaaJoseph** [comment]
Still isn't fixed for me, using version 0.9.0

**Comment by plfiorini** [comment]
@AlaaJoseph A new release has not been released yet. Did you try with the develop branch?
